src/lib/heroku-log-drain-check.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * @typedef {import("../../typings/n-express").HerokuLogDrainHealthcheckOptions} HerokuLogDrainHealthcheckOptions
+ * @typedef {import("../../typings/metrics").Healthcheck} Healthcheck
+ * @typedef {import("../../typings/metrics").TickingMetric} TickingMetric
+ */
+const nHealth = require('n-health');
+
+/**
+ * @param {HerokuLogDrainHealthcheckOptions?} [opts]
+ * @returns {Healthcheck & TickingMetric}
+ */
+module.exports = (opts) => {
+	opts = opts || {};
+
+	const region = process.env.REGION === 'US' ? 'us' : 'eu';
+
+	return nHealth.runCheck({
+		id: `heroku-log-drain-${region}`,
+		name: 'Heroku log drain configured',
```

Comment:
__nitpick:__ Would you be happy to have something in the name which differentiates the checks per region? The app name would be cool but I don't think we have that here. So maybe just the region.

Otherwise we end up with multiple checks in Heimdall with the same name which are hard to distinguish.